packages/instrumentation-aws-sdk/test/dynamodb.spec.ts

```diff
@@ -69,4 +66,57 @@ describe('DynamoDB', () => {
             });
         });
     });
+
+    describe('BatchGetIem', () => {
+        beforeEach(() => {
+            mockAwsSend(responseMockSuccess, {
+                Responses: { 'test-table': [{ key1: { S: 'val1' } }] },
+                UnprocessedKeys: {},
+                ConsumedCapacity: [
+                    {
+                        TableName: 'test-table',
+                        CapacityUnits: 0.5,
+                        Table: { CapacityUnits: 0.5 },
+                    },
+                ],
+            } as AWS.DynamoDB.Types.BatchGetItemOutput);
+            instrumentation.disable();
+            instrumentation.enable();
+        });
+        it('should populate specific BatchGetIem attributes', (done) => {
+            const dynamodb = new AWS.DynamoDB.DocumentClient();
+            const dynamodb_params = {
+                RequestItems: {
+                    'test-table': {
+                        Keys: [{ key1: { S: 'val1' } }],
+                        ProjectionExpression: 'id',
+                    },
+                },
+                ReturnConsumedCapacity: 'INDEXES',
+            };
+            dynamodb.batchGet(
+                dynamodb_params,
+                (err: AWSError, data: AWS.DynamoDB.DocumentClient.BatchGetItemOutput) => {
+                    const spans = getTestSpans();
+                    expect(spans.length).toStrictEqual(1);
+                    const attrs = spans[0].attributes;
+                    expect(attrs[SemanticAttributes.DB_SYSTEM]).toStrictEqual('dynamodb');
+                    expect(attrs[SemanticAttributes.DB_OPERATION]).toStrictEqual('BatchGetItem');
+                    expect(attrs[SemanticAttributes.AWS_DYNAMODB_TABLE_NAMES]).toStrictEqual(['test-table']);
+                    expect(attrs[SemanticAttributes.AWS_DYNAMODB_CONSUMED_CAPACITY]).toStrictEqual(
+                        [
```

Comment:
Optional: since we use the exact same object for both the mock response and the assertion, we can extract it into a variable. That will signal the source of this value to the reader of the code

Reply:
Thanks I think that's a great idea! 😄 Changes made!